SparkPost/2web2ui
# FE-1079 - Within table rows, <Button></Button> components should have icons and use the flat, gray color

[FE-1079](https://jira.int.messagesystems.com/browse/FE-1079)

### What Changed
- Adds new `"minimal"` variant to the `Button` component
- Increases `z-index` by 1 for the OG Modal - this helps with overlay problem when opening a modal via a `Popover`
- Adds screen reader only content within empty table headings
- Updates buttons within tables in the app:
  + When more than one `Button` is present, uses the `ActionPopover` component
  + When only one `Button` is present, uses the `"minimal"` variant paired with an icon on the righthand side
- Updates tests accordingly

### How To Test
- Verify all buttons within tables were updated according to the a/c
- Verify that existing behavior within these tables continues to function as expected

### To Do
